website/templates/project/registration_editor_templates.mako

```diff
@@ -71,7 +71,7 @@
 
 <script type="text/html" id="object">
   <span data-bind="foreach: $data.properties">
-    <p class="help-block breaklines f-w-xl" data-bind="text: $data.description"></p>
+    <p class="help-block breaklines f-w-xl" data-bind="html: $data.description"></p>
```

Comment:
Will user-entered data ever appear for `data.description`?

Reply:
Nope. It's the description from the shema we controlled. The reason for this is that RIDIE want to show help link in the description to their website.

Reply:
OK. That's what I thought. Thanks for verifying.